babel/babel.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/sh
 pybabel extract -F babel.ini -k _gettext -k _ngettext -k lazy_gettext -o admin.pot --project Flask-Admin ../flask_admin
+
+# Uncomment below if new messages need to be pulled in
+# pybabel update -i admin.pot -d ../flask_admin/translations -D admin -N
```

Comment:
Wouldn't it make more sense to do this via CLI parameter? E.g.:
```sh
[ "$1" = '--update' ] && pybabel update -i admin.pot -d ../flask_admin/translations -D admin -N
```

Reply:
Yes 👌